### DISCUSSION TITLE
Mentioning the keyboard layout in the Dasharo Tools Suite docs

### DISCUSSION THREAD

**Discussion by wessel-novacustom** [Dasharo/docs#780] [Ideas]
Many of our customers don't use the US keyboard layout. I think the docs overview documentation page should have a warning that only the US keyboard layout is supported in the EDK II environment.

**Comment by miczyg1** [comment]
@macpijan @mkopec cc

**Comment by macpijan** [comment]
I assume you mean this page: https://docs.dasharo.com/dasharo-menu-docs/overview/
Somewhere at the top of this document
